lib/v2/screens/profile_screens/profile/profile_screen.dart

```diff
@@ -27,14 +30,40 @@ class ProfileScreen extends StatelessWidget {
               case PageState.failure:
                 return const FullPageErrorIndicator();
               case PageState.success:
-                return ListView(
-                  children: <Widget>[
-                    const ProfileHeader(),
-                    const DividerJungle(thickness: 2),
-                    const ProfileMiddle(),
-                    const DividerJungle(thickness: 2),
-                    const ProfileBottom(),
-                  ],
+                return Scaffold(
+                  appBar: AppBar(
+                    title: BlocBuilder<ProfileBloc, ProfileState>(builder: (context, state) {
+                      return DropdownButton<bool>(
+                        value: true,
+                        icon: const Icon(Icons.keyboard_arrow_down_sharp, color: AppColors.white),
```

Comment:
do we need to use white here? Isnt white default ?

Reply:
Yes, at the moment I have to overwrite it because when it is disabled (no items) as its is right now, the dropdown shows the icon with grey color.


Here I only seek to fulfill the design of Figma because it is not yet clear to me if this is really a dropdown or a text with an icon next to it. I need clarify this behaviour with Jason yet.